source/manual/data-gov-uk-troubleshooting.html.md

```diff
@@ -68,3 +68,33 @@ If you see a lot of tracebacks, it might be necessary to restart Celery.
 ```
 sudo initctl restart celery_bulk-procfile-worker
 ```
+
+## Harvesters not processing or seem stuck
+
+The harvesting process runs as a single threaded program, if any harvesting
+process crashes by raising an exception, it will take out the entire process.
+We have configured Upstart to restart the process automatically, but if the
+service keeps crashing, Upstart will decide it's unhealthy and stop that after
+a while.
+
+You can check whether the process is still running by checking if entries are
+still being written to the log file:
```

Comment:
Could we pop the machine class we should be sshing into in this sentence just to reduce mental overhead a bit. It's implied in the fab script down the bottom, but nice to have up here as well.

Reply:
Good idea, I've added a bit about that to this section and the section above.